apps/frontend/src/components/Reviews.tsx

```diff
@@ -44,21 +60,46 @@ const Reviews = ({ reviews }: { reviews: Review[] }) => {
       return;
     }
 
-    const newReviewObject = {
-      listing_rating_id: rating,
+    const newReviewObject: NewReview = {
+      listing_review_id: new Date().getTime().toString(), // Temporary ID for client-side only
       stars: rating,
       comment: reviewText,
-      listingId: reviews[0].listingID,
+      listingID: listingID,
     };
 
     try {
-      await _axios_instance.post("/review", newReviewObject);
+      const response = await _axios_instance.post("/review", newReviewObject);
+      const reviewId = response.data.review_id;
+      const fullReviewObject: Review = {
+        ...newReviewObject,
+        listing_review_id: reviewId,
+        reviewerName: user?.username || "Anonymous",
+        userID: user?.userID || "CurrentUser",
+        dateCreated: new Date().toISOString(),
```

Comment:
Date created and date modified should be set on the backend instead of here, but this is fine for now

Reply:
Updated this so it sets dateCreated and dateModified from backend